src/dotnetcore/integration/sealights_test.go

```diff
@@ -1,59 +1,59 @@
-package integration_test

```

Comment:
@brayanhenao What's the diff in this file? I can't figure out.
Same for `src/dotnetcore/hooks/sealights.go`

Reply:
@arjun024 That's weird, maybe LF format?

Reply:
Yes `git show --ignore-space-at-eol -- src/dotnetcore/integration/sealights_test.go` is empty.
Can you unstage both of those files?

Reply:
@arjun024 Done!